pkg/subctl/cmd/join.go

```diff
@@ -100,6 +102,8 @@ func addJoinFlags(cmd *cobra.Command) {
 		"GlobalCIDR to be allocated to the cluster")
 	cmd.Flags().StringSliceVar(&customDomains, "custom-domains", nil,
 		"List of domains to use for multicluster service discovery")
+	cmd.Flags().StringSliceVar(&imageOverrideArr, "image-overrides", nil,
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&imageOverrideArr, "image-override", nil,
```

Reply:
> To override one or multiple component images we can use the flag
>     --image-override="<component1>=<image_path1>,<component2>=<image_path2>,..."
>     as part of subctl join.
> 
> Wouldn't this be --image-override =<image_path1> --image-override = ?
> 
>     Closes #830
> 
>     Signed-off-by: Steve Mattar smattar@redhat.com

@mangelajo  according to cobra Flags().StringSliceVar both should work

Reply:
@SteveMattar @mangelajo I think following  --image-override =<image_path1> --image-override =<image_path2>  would be easier as you only need to know the image you will override and not the component

Reply:
That's true, that would be far more usable,

but it would make the detection logic more twisted, in some cases (downstream) we well have prefixes/subfixes

We can always improve it later and try to add auto-component-detection?

Reply:
@mangelajo I think Carlos has a point. If the middle-name is a constant then I can easily detect the component. No need to complicate the user.